packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -130,28 +130,77 @@ class OnsetPicker extends React.Component {
       onsetDate: this.props.onsetDate ? moment(this.props.onsetDate) : undefined,
       age: ageValues.age,
       ageUnit: ageValues.ageUnit,
+      year: undefined,
+      month: undefined,
     };
   }
 
+  componentDidMount() {
+    if (this.props.onsetDate) {
+      const firstDayOfYear = moment().year(moment(this.props.onsetDate).year()).startOf('year').format(DATE_FORMAT);
+      const firstDayOfMonth = moment().year(moment(this.props.onsetDate).year()).month(moment(this.props.onsetDate).month()).startOf('month')
+        .format(DATE_FORMAT);
+      if (this.props.granularity === 'year' && this.props.onsetDate === firstDayOfYear) {
```

Comment:
for consistency `'year'` should be replaced by  `GranularityOptions.YEAR`.

Reply:
Changed c4b4f2f